apps/api/src/app/events/usecases/parse-event-request/parse-event-request.usecase.ts

```diff
@@ -137,7 +137,14 @@ export class ParseEventRequest {
       transactionId,
     };
 
-    await this.workflowQueueService.add({ name: transactionId, data: jobData, groupId: command.organizationId });
+    const options = command.priority ? { priority: command.priority } : {};
```

Comment:
Leveraging BullMQ priorities makes sense in a self hosted setup but not in a cloud deployment as jobs are multiplexed across different organizations.

I suggest adding a `if (process.env.IS_DOCKER_HOSTED === 'true') {` check before passing the priority option.

Reply:
Done